### PR TITLE
backport: trivial 2023 12 25

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
## What was done?
Small batch of trivial backports

## How Has This Been Tested?
make -j8 check; functional tests; linters

## Checklist:
  _Go over all the following points, and put an `x` in all the boxes that apply._
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation
- [x] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_

### CODE REVIEW

File: src/protocol.h [hunk 1]
```diff
@@ -479,7 +479,7 @@ class CAddress : public CService
         SerReadWriteMany(os, ser_action, ReadWriteAsHelper<CService>(obj));
     }
 
-    //! Always included in serialization, except in the network format on INIT_PROTO_VERSION.
+    //! Always included in serialization.
```
Comment (knst) [inline src/protocol.h:482]: that's follow-up changes for https://github.com/bitcoin/bitcoin/pull/20541 which is not backported yet.

File: doc/multisig-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+Currently, it is possible to create a multisig wallet using Bitcoin Core only.
+
+Although there is already a brief explanation about the multisig in the [Descriptors documentation](https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md#multisig), this tutorial proposes to use the signet (instead of regtest), bringing the reader closer to a real environment and explaining some functions in more detail.
+
+This tutorial uses [jq](https://github.com/stedolan/jq) JSON processor to process the results from RPC and stores the relevant values in bash variables. This makes the tutorial reproducible and easier to follow step by step.
+
+Before starting this tutorial, start the bitcoin node on the signet network.
+
+```bash
+./src/bitcoind -signet -daemon
```
Comment (knst) [inline doc/multisig-tutorial.md:12]:
```suggestion
./src/dashd -testnet -daemon
```

File: doc/multisig-tutorial.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
+# 1. Multisig Tutorial
+
+Currently, it is possible to create a multisig wallet using Bitcoin Core only.
+
+Although there is already a brief explanation about the multisig in the [Descriptors documentation](https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md#multisig), this tutorial proposes to use the signet (instead of regtest), bringing the reader closer to a real environment and explaining some functions in more detail.
+
+This tutorial uses [jq](https://github.com/stedolan/jq) JSON processor to process the results from RPC and stores the relevant values in bash variables. This makes the tutorial reproducible and easier to follow step by step.
+
+Before starting this tutorial, start the bitcoin node on the signet network.
```
Comment (knst) [inline doc/multisig-tutorial.md:9]:
```suggestion
Before starting this tutorial, start the bitcoin node on the testnet network.
```

File: doc/multisig-tutorial.md [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+
+```bash
+./src/bitcoind -signet -daemon
+```
+
+This tutorial also uses the default WPKH derivation path to get the xpubs and does not conform to [BIP 45](https://github.com/bitcoin/bips/blob/master/bip-0045.mediawiki) or [BIP 87](https://github.com/bitcoin/bips/blob/master/bip-0087.mediawiki).
```
Comment (knst) [inline doc/multisig-tutorial.md:15]: `wpkh` is witness pkh.

Let's drop this backport https://github.com/bitcoin/bitcoin/pull/22822 so far as it requires non-trivial dashification?

**Review by knst — CHANGES_REQUESTED** [review]
please drop (or dashificate) https://github.com/bitcoin/bitcoin/pull/22822

Probably drop https://github.com/bitcoin/bitcoin/pull/22780 also (see comment)

**Review by knst — APPROVED** [review]
utACK

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**